sdcm/tester.py

```diff
@@ -426,11 +426,13 @@ def argus_get_scylla_version(self):
             self.log.info("Collection Scylla version for argus...")
             version_regex = re.compile(r'(([\w.~]+)-(0.)?([0-9]{8,8}).(\w+).)')
             version_str = self.db_cluster.nodes[0].get_scylla_binary_version()
-            if (match := version_regex.match(version_str)):
+            if version_str and (match := version_regex.match(version_str)):
                 self.argus_test_run.run_info.details.scylla_version = match.group(2)
                 self.argus_test_run.save()
+            return
         except Exception:  # pylint: disable=broad-except
             self.log.error("Error getting scylla version for argus", exc_info=True)
+        raise CriticalTestFailure("Failed to get scylla version for argus")
```

Comment:
Don't we have Error? why critical?

Reply:
I've replaced it with just `Exception` for now, don't see other classes used for this in tester, but I see that unit tests are failing now, so maybe will have to patch this function out of the unit tests.

Reply:
we should raise an event here, if we don't want to stop the test.

any exception here would stop the test (regardless of it's name), it's part of the test setup, and if it fails the test would won't run.